rllib/BUILD

```diff
@@ -512,6 +544,40 @@ py_test(
     srcs = ["tuned_examples/impala/multi_agent_cartpole_impala.py"],
     args = ["--as-test", "--enable-new-api-stack", "--num-agents=2", "--num-gpus=2", "--num-cpus=7"]
 )
+# StatelessCartPole
+py_test(
+    name = "learning_tests_stateless_cartpole_impala",
+    main = "tuned_examples/impala/stateless_cartpole_impala.py",
+    tags = ["team:rllib", "exclusive", "learning_tests", "torch_only", "learning_tests_discrete", "learning_tests_pytorch_use_all_core"],
+    size = "large",
+    srcs = ["tuned_examples/impala/stateless_cartpole_impala.py"],
+    args = ["--as-test", "--enable-new-api-stack", "--num-gpus=1"]
```

Comment:
And here. I guess this brings us a `num_learners=1`, doesn't it?

Reply:
This actually tries to put the 1 (remote) Learner on 1 GPU.

Sorry, you are right in that these command line options are very confusing:

On a CPU machine:
`--num-gpus=1` -> 1 (remote) Learner (on CPU!)
`--num-gpus=2` -> 2 (remote) Learners (on CPUs!)

On a GPU machine:
`--num-gpus=1` -> 1 (remote) Learner (on GPU)
`--num-gpus=2` -> 2 (remote) Learners (on GPUs)

We should probably rename these args.